### DISCUSSION TITLE
Sega Megadrive Mini JP dat file

### DISCUSSION THREAD

**Discussion by mauruco** [farmerbb/RED-Project#41] [General]
[Sega Megadrive Mini JP.txt](https://github.com/farmerbb/RED-Project/files/8633726/Sega.Megadrive.Mini.JP.txt)
Hello guys. I really like your work.
I recently purchased a Sega Mini JP. I saw that the link https://github.com/farmerbb/RED-Project/wiki/Sega-Genesis-Mini doesn't have the dat-file for the Japanese version of the mini. So I created one and maybe you guys want to add it.
Thank you very much.

**Comment by farmerbb** [comment]
Added to the repo and also to the wiki page. Thanks!
